checked_message.go

```diff
@@ -20,14 +20,21 @@
 
 package zap
 
-import "github.com/uber-go/atomic"
+import "sync"
+
+var _CMPool = sync.Pool{
```

Comment:
I *think* that the typical rule for initialisms would make this `_cmPool`. https://github.com/golang/go/wiki/CodeReviewComments#initialisms